paddle/fluid/framework/tensor_util.cc

```
@@ -495,6 +495,127 @@ void TensorFromStream(std::istream& is, Tensor* tensor,
   }
 }
 
+// get tensor data point by DLDataType
+void* GetDstPtrByDLDataType(DLDataType type, framework::Tensor* dst,
+                            const platform::CPUPlace& dst_place) {
+  if (type.lanes > 1) return nullptr;  // vector types not currently supported
```

Throw exception instead of hide errors when `type.lanes > 1`?

---

Done